src/Dofs/apply_analytical.jl

```diff
@@ -0,0 +1,114 @@
+
+function _default_interpolations(dh::MixedDofHandler)
+    fhs = dh.fieldhandlers
+    getcelltype(i) = typeof(getcells(dh.grid, first(fhs[i].cellset)))
+    ntuple(i->default_interpolation(getcelltype(i)), length(fhs))
+end
+
+function _default_interpolation(dh::DofHandler)
+    return default_interpolation(typeof(getcells(dh.grid, 1)))
+end
+
+# If fieldname==nothing, check that dh only has one field and return that fieldname 
+# If a fieldname is given, check that it exist in the dofhandler
+function _check_fieldname(dh, fieldname)
```

Comment:
Might be easier to always require to specify? Same as e.g. Dirichlet BC etc

Reply:
Makes sense!